alot/account.py

```diff
@@ -20,6 +20,7 @@ class StoreMailError(Exception):
 
 
 class Account(object):
+
```

Comment:
The templates I use when writing new code have that so I'm used to it... Will remove them.